### PR TITLE
Hide direction if the address is empty

### PR DESCRIPTION
**Pull request by shuklina**
PR https://github.com/ymcatwincities/yn/pull/5797
image

- [ ] Make sure that if there is no address for some branch, directions, and address are not displayed in the block. Path `/locations`

### CODE REVIEW

**Review by podarok — APPROVED** [review]

File: openy_location/modules/openy_loc_branch/config/optional/field.field.node.branch.field_location_address.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+langcode: en
```
Comment (podarok) [inline openy_location/modules/openy_loc_branch/config/optional/field.field.node.branch.field_location_address.yml:1]: this config should go to original place https://github.com/open-y-subprojects/openy_features/blob/main/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_address.yml

File: openy_location/modules/openy_loc_branch/config/optional/field.field.node.branch.field_location_directions.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+langcode: en
```
Comment (podarok) [inline openy_location/modules/openy_loc_branch/config/optional/field.field.node.branch.field_location_directions.yml:1]: this should be in original place https://github.com/open-y-subprojects/openy_features/blob/main/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_directions.yml

**Review by podarok — CHANGES_REQUESTED** [review]

File: openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_directions.yml [hunk 3]
```diff
@@ -11,17 +11,19 @@ third_party_settings:
   datalayer:
     expose: 1
     label: field_location_directions
+_core:
+  default_config_hash: wECsSrrpKwpR9l0_0o0esc5NK7Rf-JTr46G-JOhFzEU
```
Comment (podarok) [inline openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_directions.yml:15]: let's remove `_core` section completely

**Review by podarok — CHANGES_REQUESTED** [review]

**Comment by shelleygoetz** [comment]
Link to Jira ticket: https://openy.atlassian.net/browse/MAINTAIN-256

**Pull request merged**